deployment/cdk/opensearch-service-migration/README.md

```diff
@@ -175,16 +181,17 @@ With the [required setup](#importing-target-clusters) on the target cluster havi
 ./accessContainer.sh migration-console dev us-east-1
 ```
 
-* Execute the ECS run task command stored in the container's environment
+* Execute the ECS run task command generated by `showFetchMigrationCommand.sh` script. 
     * The status of the ECS Task can be monitored from the AWS Console. Once the task is in the `Running` state, logs and progress can be viewed via CloudWatch.
 ```shell
-# This will print the needed ECS command
-echo $FETCH_MIGRATION_COMMAND
+# This will execute the script and print the required ECS run task command
+./showFetchMigrationCommand.sh
 
-# Paste the above printed command into the terminal to kick off
+# Paste command output by the script into the terminal to kick off Fetch Migration 
 ```
 
 The pipeline configuration file can be viewed (and updated) via AWS Secrets Manager.
+Please note that it will be base64 encoded.
```

Comment:
Thanks for updating this, did you need to update the wiki as well?

Reply:
I did not, but I just took a look at it again and it doesn't mention Fetch Migration, so I'll be adding a section on it.